### PR TITLE
Fix code style and add copyright.

### PR DESCRIPTION
**Pull request by pkufool**
@csukuangfj I disabled the E501 rule of flake8 for file `egs/librispeech/ASR/conformer_ctc/conformer.py` as it will cut the comments into annoying pieces, especially the example code.

### CODE REVIEW

File: egs/librispeech/ASR/conformer_ctc/pretrained.py [hunk 1]
```diff
@@ -248,6 +264,7 @@ def main():
         G = G.to(device)
         # Add epsilon self-loops to G as we will compose
         # it with the whole lattice later
+        G = G.to(device)
```
Comment (csukuangfj) [inline egs/librispeech/ASR/conformer_ctc/pretrained.py:267]:
```suggestion
```

**Comment by csukuangfj** [comment]
The following is how the CI checks the code style and format:
https://github.com/k2-fsa/icefall/blob/8469f9ae0a2d7686f04e558fba8ddfb5505109fc/.github/workflows/style_check.yml#L46-L62

Could you fix the black version to the one mentioned in
https://github.com/k2-fsa/icefall/blob/8469f9ae0a2d7686f04e558fba8ddfb5505109fc/.pre-commit-config.yaml#L2-L6

i.e.
```
python3 -m pip install --upgrade pip black==21.6b0  flake8 
```

---

https://github.com/k2-fsa/icefall/blob/master/contributing.md
says how to set up the dev environment so that you can check the style locally.

File: egs/librispeech/ASR/conformer_ctc/conformer.py [hunk 2]
```diff
@@ -15,18 +29,18 @@
...
-        d_model (int): attention dimension
-        nhead (int): number of head
-        dim_feedforward (int): feedforward dimention
-        num_encoder_layers (int): number of encoder layers
-        num_decoder_layers (int): number of decoder layers
-        dropout (float): dropout rate
-        cnn_module_kernel (int): Kernel size of convolution module
-        normalize_before (bool): whether to use layer_norm before the first block.
-        vgg_frontend (bool): whether to use vgg frontend.
+      num_features (int): Number of input features
```
Comment (csukuangfj) [inline egs/librispeech/ASR/conformer_ctc/conformer.py:32]: Did you reformat this file by hand or using `black`?

I just ran black on this file and nothing is changed.

(I am using black==21.6b0)

Comment (pkufool) [reply egs/librispeech/ASR/conformer_ctc/conformer.py]: I reverted the reformatting of conformer.py. It seems some conflicts between black and flake8, so ignored the E203 checking for that line.
image

**Pull request merged**